docs/website/src/components/examples.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable node/no-missing-import */
+import React, { useState } from 'react'
+
+import ArrowIcon from '../../static/arrow-icon.svg'
+import GithubIcon from '../../static/github-icon.svg'
+
+const GithubLink = ({ link }) => (
+  <a
+    href={link}
+    class='group-hover:text-white w-150 -ml-1 p-1 rounded-md text-left hover:bg-blue-500 text-gray-700 align-middle cursor-pointer'
+  >
+    <span class='float-left mt-1 no-underline'>View Source</span>
+    <GithubIcon
+      alt='View Source'
+      class='float-right mx-2 h-8 w-8 fill-current'
+    />
+  </a>
+)
+
+const createRootElement = () => {
+  const root = document.createElement('div')
+  const id = 'root'
```

Comment:
id='root' is usually for the main React app in the page. In this case it would be the docusaurus app, but I guess they use another name (otherwise you'd get an error in the console).
I prefer to change this name to something more custom like 'webchatExampleRoot'

Reply:
sure, i would change it or generate it randomly to avoid this kind of potential issues